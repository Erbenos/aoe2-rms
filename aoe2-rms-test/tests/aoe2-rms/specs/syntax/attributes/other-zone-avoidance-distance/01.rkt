#lang aoe2-rms
; A rivers map where all players are separated by water, and there is a neutral island in the center.

<LAND-GENERATION>
(base-terrain 'WATER)
(create-player-lands
 (terrain-type 'GRASS)
 (land-percent 100)
 (other-zone-avoidance-distance 10))

(create-land
 (terrain-type 'DIRT)
 (land-percent 100)
 (land-position 50 50)
 (zone 1)
 (other-zone-avoidance-distance 10))
