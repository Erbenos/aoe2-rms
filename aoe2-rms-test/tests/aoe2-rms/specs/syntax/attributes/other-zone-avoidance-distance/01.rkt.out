<LAND_GENERATION>
base_terrain WATER
create_player_lands {
  terrain_type GRASS
  land_percent 100
  other_zone_avoidance_distance 10
}
create_land {
  terrain_type DIRT
  land_percent 100
  land_position 50 50
  zone 1
  other_zone_avoidance_distance 10
}
